src/zjs_fs.c

```diff
@@ -488,59 +467,23 @@ static jerry_value_t zjs_write(const jerry_value_t function_obj,
                                const jerry_length_t argc,
                                uint8_t async)
 {
+    // args: file descriptor, buffer, offset, length[, position]
+    ZJS_VALIDATE_ARGS_OPTCOUNT(optcount, Z_OBJECT, Z_OBJECT, Z_NUMBER,
+                               Z_NUMBER, Z_OPTIONAL Z_NUMBER);
```

Comment:
The Node prototype for this is: `write(fd, buffer[, offset[, length[, position]]])`. Can we put Z_OPTIONAL in front of all the needed parameters? I tried it locally and it "worked" i.e. test-fs.js passed. Whether or not it actually validated I am not sure but if the validation can't do multiple optional arguments then we need to change the docs/test for FS because it explicitly tests different ways of calling fs.writeSync().